external-crates/move/crates/move-vm-runtime/src/shared/constants.rs

```diff
@@ -9,6 +9,9 @@
 pub const OPERAND_STACK_SIZE_LIMIT: usize = 1024;
 pub const CALL_STACK_SIZE_LIMIT: usize = 1024;
 
+/// Maximal number of locals any individual call can have.
+pub const LOCALS_PER_FRAME_LIMIT: usize = 2_048;
```

Comment:
<img src="https://media1.giphy.com/media/84BjZMVEX3aRG/giphy.gif"/>

plz remove the `_`...